src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -40,16 +54,29 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+    
+  <ItemGroup>
+    <None Update="oras-OSX">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
   
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
     <Exec Command="mkdir ..\..\bin &amp; tar -zcvf ../../bin/Hl7v2DefaultTemplates.tar.gz -C ../../data/Templates/Hl7v2 ." />
     <Exec Command="tar -zcvf ../../bin/CcdaDefaultTemplates.tar.gz -C ../../data/Templates/Ccda ." />
   </Target>
   
-  <Target Name="DownloadOrasFile" BeforeTargets="Build">
+  <Target Name="DownloadWinOrasFile" BeforeTargets="Build">
     <DownloadFile SourceUrl="$(OrasWinUrl)" DestinationFolder="../../bin">
       <Output TaskParameter="DownloadedFile" ItemName="Content" />
     </DownloadFile>
-    <Exec Command="tar -xvf ..\..\bin\oras_0.8.1_windows_amd64.tar.gz -C ..\..\bin &amp; copy ..\..\bin\oras.exe ." />
+    <Exec Command="tar -xvf $(BinFolderPath)oras_0.8.1_windows_amd64.tar.gz -C $(BinFolder) &amp; $(CopyExecutable) $(BinFolderPath)oras.exe ." />
```

Comment:
Would it better to use $(OrasVersion) instead of 0.8.1?

Reply:
fixed it, thanks.